packages/tupaia-web/src/api/queries/useEntities.ts

```diff
@@ -2,16 +2,27 @@
  * Tupaia
  *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
  */
-
-import { useQuery } from 'react-query';
+import { AxiosRequestConfig } from 'axios';
+import { useQuery, QueryObserverOptions, QueryOptions } from 'react-query';
+import { Entity } from '@tupaia/types';
 import { get } from '../api';
 
-export const useEntities = (projectCode: string, entityCode?: string, options?: any) => {
+type EntityResponse = Entity & { children?: Entity[] };
```

Comment:
This feels like a type that might be useful down the track, so could go in the `types` file. Not a biggie, just a thought

Reply:
Well Ethan has a ticket to generate all the response types for the orchestration server so I'm hoping this one can just be swapped out when that is ready.